### PR TITLE
fix(userspace/plugin/plugin_types.h): add missing ss_plugin_field_type

### PR DESCRIPTION
**Pull request by therealbobo**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

> /area driver-kmod

> /area driver-bpf

> /area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

> /area libscap-engine-savefile

> /area libscap-engine-udig

> /area libscap

> /area libpman

> /area libsinsp

> /area tests

> /area proposals

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

> /version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

**What this PR does / why we need it**:

The new `ss_plugin_field_type` added in the plugin-sdk-go were missing in libs.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/plugin/plugin_types.h [hunk 1]
```diff
@@ -31,8 +31,15 @@ typedef uint32_t ss_plugin_bool;
 // falcosecurity libs types.
 typedef enum ss_plugin_field_type
 {
-	FTYPE_UINT64 = 8,
-	FTYPE_STRING = 9
+	FTYPE_UINT64      = 8,
+	FTYPE_STRING      = 9,
+	FTYPE_IPV4ADDR    = 12,
```
Comment (Andreagit97) [inline userspace/plugin/plugin_types.h:36]: what is the reason behind these numbers? at first glance, it seems they are matching the `ppp_param_type` enum but `PT_IPV4ADDR` is 26 while here is 12 :thinking:

Comment (therealbobo) [reply userspace/plugin/plugin_types.h]: Ops! My bad. I'll fix it also in the sdk. The comment it's already there:
https://github.com/falcosecurity/libs/blob/1299cb4de1515d2aa5dc8d636787be728f127779/userspace/plugin/plugin_types.h#LL30C1-L30C1

**Review by Andreagit97 — APPROVED** [review]
/approve 
thank you!

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**